sensing/position_estimation.hpp

```diff
@@ -201,10 +201,13 @@ class Position_estimation: public INS
 
     float gravity;                          ///< Value of the gravity
 
+    float barometer_bias;
+    bool barometer_calibrated;
+
     const ahrs_t& ahrs;                     ///< Reference to the attitude estimation structure
     State& state;                           ///< Reference to the state structure
     const Gps& gps;                         ///< Reference to the GPS structure
-    Barometer& barometer;                   ///< Reference to the barometer structure
+    const Barometer& barometer;                   ///< Reference to the barometer structure
```

Comment:
done in https://github.com/lis-epfl/MAVRIC_Library/pull/311/commits/2f813793cac5e2ebf2a7fd5f365261eca57dc003